torch/nn/modules/module.py

```diff
@@ -1287,6 +1289,8 @@ def eval(self: T) -> T:
 
         This is equivalent with :meth:`self.train(False) <torch.nn.Module.train>`.
 
+        Additionally, consider using :class:`~no_grad` when evaluating a model.
```

Comment:
Like above, I would add a bit more detail here like: `, to de-activate the autograd.`